Source/DafnyDriver/CliCompilation.cs

```diff
@@ -22,15 +22,18 @@ namespace DafnyDriver.Commands;
 public record CanVerifyResult(ICanVerify CanVerify, IReadOnlyList<VerificationTaskResult> Results);
 
 public class CliCompilation {
+  private readonly ILogger<CliCompilation> logger;
```

Comment:
This looks like it's never used. Am I missing something?

Reply:
Unused indeed. Usage only occurred in a previous commit. Removed it.